Application/src/Utils/Structure.js

```diff
@@ -1,64 +1,62 @@
 import { Image } from 'image-js';
 import { changeDpiDataUrl } from 'changedpi';
 import { getText } from './Text';
-import { getCardHeaderUtilityClass } from '@mui/material';
 
 export async function getImageGrid(image) {
   const ogImage = await Image.load(image);
   const { width, height } = ogImage;
   let newImages = {
+    tempImages: {
+      top: {
+        img: null,
+        text: null,
+        textConfidence: null,
+      },
+      middle: {
+        img: null,
+        text: null,
+        textConfidence: null,
+      },
+      bottom: {
+        img: null,
+        text: null,
+        textConfidence: null,
+      },
+    },
```

Comment:
Can this mega object be divided into two separate objects?

Reply:
Yes, I will split them.